growth/too/tasks/ref.py

```diff
@@ -0,0 +1,49 @@
+from astropy import table
+from pyvo.dal import TAPService
+from celery.task import PeriodicTask
+from celery.utils.log import get_task_logger
+
+from . import celery
+from .. import models
+
+log = get_task_logger(__name__)
+
+__all__ = ('ztf_references',)
+
+
+def get_tap_client():
+    url = 'https://irsa.ipac.caltech.edu/TAP'
+    return TAPService(url)
+
+
+@celery.task(base=PeriodicTask, shared=False, run_every=3600)
+def ztf_references():
+
+    refstable = get_tap_client().search("""
+    field, ccdid, qid, fid, maglimit FROM ztf.ztf_current_meta_ref
+    WHERE nframes >= 15 AND startobsdate >= '2018-02-05T00:00:00Z'
+    """).to_table()
+
+    refs = table.unique(refstable, keys=['field', 'fid'])
+
+    reference_images = \
```

Comment:
Suggestion:

```python
refs_grouped_by_field = refutable.group_by('field').groups
for field_id, rows in zip(refs_grouped_by_field.keys, refs_grouped_by_field):
    models.db.session.merge(
        models.Field(
            telescope='ZTF', field_id=field_id,
            reference_filter_ids=rows['fid'].tolist(),
            reference_filter_mags=rows['maglimit'].tolist()))
```

Reply:
When doing rows['fid'].tolist(), I get: raise ValueError('Unable to compare rows for masked table due to numpy.ma bug').

Reply:
Are there actually any masked values in the table? If not, then you can [convert it to an unmasked table](http://docs.astropy.org/en/stable/table/masking.html#table-creation).